driver.go

```diff
@@ -125,6 +188,16 @@ func (c *conn) ExecContext(ctx context.Context, query string, args []driver.Name
 	return &result{rowsAffected: rowsAffected}, nil
 }
 
+func IsDdl(query string) (bool, error) {
```

Comment:
nit: if we extract a class for verifying what a query is, we could unit test this separately

Reply:
will create enum to group all statement types for next pr 🙇‍♀️